zebrad/src/components/mempool/tests.rs

```diff
@@ -219,3 +228,120 @@ async fn mempool_queue() -> Result<(), Report> {
 
     Ok(())
 }
+
+#[tokio::test]
+async fn mempool_service_disabled() -> Result<(), Report> {
```

Comment:
I think this will conflict with the code movement in PR #2765.
So it should probably move to a test submodule once that PR merges.

Reply:
That only organized code in `mempool/storage/tests.rs` and not `mempool/tests.rs`, so it should be fine.